airflow_tecton/hooks/tecton_hook.py

```diff
@@ -155,15 +158,16 @@ def find_materialization_job(
         for job in sorted(
             jobs, reverse=True, key=lambda x: self._parse_time(x["created_at"])
         ):
-            if job["job_type"].lower() != "batch":
-                continue
-            if (
-                job["online"] == online
-                and job["offline"] == offline
-                and job["start_time"] == self._canonicalize_datetime(start_time)
-                and job["end_time"] == self._canonicalize_datetime(end_time)
-            ):
-                return job
+            self.log.info('job = %s' % job)
+            job_job_type = job.get("job_type", "")
```

Comment:
variable naming is confusing. maybe actual_job_type?

Reply:
sure